polaris-ui/src/app/features/cases/hooks/useNavigationAlert.ts

```diff
@@ -51,7 +51,8 @@ export const useNavigationAlert = (
     return () => {
       navigationUnblockHandle.current && navigationUnblockHandle.current();
     };
-  }, [tabItems]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
As per discussion, try to make it so that the disable comment can be removed.

Reply:
fixed